src/renderer/components/Theme/Default.scss

```diff
@@ -228,3 +228,9 @@ img {
     height: 100%;
   }
 }
+
+/* Preferences - Basic
+-------------------------- */
+.el-radio-group-margin-left {
```

Comment:
The naming of this class name... improved...

Reply:
The class move to Basic.vue and rename class name.